diff -up wxWidgets-3.1.4/build/bakefiles/wx.bkl.1~ wxWidgets-3.1.4/build/bakefiles/wx.bkl
--- wxWidgets-3.1.4/build/bakefiles/wx.bkl.1~	2020-07-22 19:20:06.000000000 +0200
+++ wxWidgets-3.1.4/build/bakefiles/wx.bkl	2020-07-23 18:58:29.549872125 +0200
@@ -137,7 +137,7 @@
         <using module="gettext"/>
         <gettext-catalogs id="locale">
             <srcdir>$(SRCDIR)/locale</srcdir>
-            <catalog-name>wxstd</catalog-name>
+            <catalog-name>wxstd-$(WX_RELEASE)</catalog-name>
             <linguas>
                 ca cs da de el es fi fr hu id it ja nl pl ru sl sv tr uk
                 zh zh_CN zh_TW
@@ -146,7 +146,7 @@
         </gettext-catalogs>
         <gettext-catalogs id="locale_msw">
             <srcdir>$(SRCDIR)/locale/msw</srcdir>
-            <catalog-name>wxmsw</catalog-name>
+            <catalog-name>wxmsw-$(WX_RELEASE)</catalog-name>
             <linguas>it</linguas>
             <install-to>$(LOCALEDIR)</install-to>
         </gettext-catalogs>
diff -up wxWidgets-3.1.4/include/wx/translation.h.1~ wxWidgets-3.1.4/include/wx/translation.h
--- wxWidgets-3.1.4/include/wx/translation.h.1~	2020-07-23 18:58:29.549872125 +0200
+++ wxWidgets-3.1.4/include/wx/translation.h	2020-07-23 18:59:30.461579578 +0200
@@ -154,7 +154,7 @@ public:
     wxString GetBestTranslation(const wxString& domain,
                                 const wxString& msgIdLanguage = wxASCII_STR("en"));
 
-    // add standard wxWidgets catalog ("wxstd")
+    // add standard wxWidgets catalog ("wxstd-3.1")
     bool AddStdCatalog();
 
     // add catalog with given domain name and language, looking it up via
diff -up wxWidgets-3.1.4/interface/wx/translation.h.1~ wxWidgets-3.1.4/interface/wx/translation.h
--- wxWidgets-3.1.4/interface/wx/translation.h.1~	2020-07-23 18:58:29.549872125 +0200
+++ wxWidgets-3.1.4/interface/wx/translation.h	2020-07-23 18:59:21.980481078 +0200
@@ -137,7 +137,7 @@ public:
                                 const wxString& msgIdLanguage = "en");
 
     /**
-        Add standard wxWidgets catalogs ("wxstd" and possible port-specific
+        Add standard wxWidgets catalogs ("wxstd-3.1" and possible port-specific
         catalogs).
 
         @return @true if a suitable catalog was found, @false otherwise
diff -up wxWidgets-3.1.4/Makefile.in.1~ wxWidgets-3.1.4/Makefile.in
--- wxWidgets-3.1.4/Makefile.in.1~	2020-07-22 19:20:06.000000000 +0200
+++ wxWidgets-3.1.4/Makefile.in	2020-07-23 18:58:29.552872160 +0200
@@ -14721,14 +14721,14 @@ locale_install:
 	$(INSTALL_DIR) $(DESTDIR)$(datadir)/locale/$$l ; \
 	$(INSTALL_DIR) $(DESTDIR)$(datadir)/locale/$$l/LC_MESSAGES ; \
 	if test -f $(srcdir)/locale/$$l.mo ; then \
-	$(INSTALL_DATA) $(srcdir)/locale/$$l.mo $(DESTDIR)$(datadir)/locale/$$l/LC_MESSAGES/wxstd.mo ; \
+	$(INSTALL_DATA) $(srcdir)/locale/$$l.mo $(DESTDIR)$(datadir)/locale/$$l/LC_MESSAGES/wxstd-$(WX_RELEASE).mo ; \
 	fi ; \
 	done
 
 locale_uninstall: 
 	for l in $(LOCALE_LINGUAS) ; do \
-	if test -f $(DESTDIR)$(datadir)/locale/$$l/LC_MESSAGES/wxstd.mo ; then \
-	rm -f $(DESTDIR)$(datadir)/locale/$$l/LC_MESSAGES/wxstd.mo ; \
+	if test -f $(DESTDIR)$(datadir)/locale/$$l/LC_MESSAGES/wxstd-$(WX_RELEASE).mo ; then \
+	rm -f $(DESTDIR)$(datadir)/locale/$$l/LC_MESSAGES/wxstd-$(WX_RELEASE).mo ; \
 	fi ; \
 	done
 
@@ -14738,14 +14738,14 @@ locale_msw_install:
 	$(INSTALL_DIR) $(DESTDIR)$(datadir)/locale/$$l ; \
 	$(INSTALL_DIR) $(DESTDIR)$(datadir)/locale/$$l/LC_MESSAGES ; \
 	if test -f $(srcdir)/locale/msw/$$l.mo ; then \
-	$(INSTALL_DATA) $(srcdir)/locale/msw/$$l.mo $(DESTDIR)$(datadir)/locale/$$l/LC_MESSAGES/wxmsw.mo ; \
+	$(INSTALL_DATA) $(srcdir)/locale/msw/$$l.mo $(DESTDIR)$(datadir)/locale/$$l/LC_MESSAGES/wxmsw-$(WX_RELEASE).mo ; \
 	fi ; \
 	done
 
 locale_msw_uninstall: 
 	for l in $(LOCALE_MSW_LINGUAS) ; do \
-	if test -f $(DESTDIR)$(datadir)/locale/$$l/LC_MESSAGES/wxmsw.mo ; then \
-	rm -f $(DESTDIR)$(datadir)/locale/$$l/LC_MESSAGES/wxmsw.mo ; \
+	if test -f $(DESTDIR)$(datadir)/locale/$$l/LC_MESSAGES/wxmsw-$(WX_RELEASE).mo ; then \
+	rm -f $(DESTDIR)$(datadir)/locale/$$l/LC_MESSAGES/wxmsw-$(WX_RELEASE).mo ; \
 	fi ; \
 	done
 
diff -up wxWidgets-3.1.4/samples/internat/internat.cpp.1~ wxWidgets-3.1.4/samples/internat/internat.cpp
--- wxWidgets-3.1.4/samples/internat/internat.cpp.1~	2020-07-22 19:20:06.000000000 +0200
+++ wxWidgets-3.1.4/samples/internat/internat.cpp	2020-07-23 18:58:29.552872160 +0200
@@ -260,7 +260,7 @@ bool MyApp::OnInit()
 
     // Now try to add wxstd.mo so that loading "NOTEXIST.ING" file will produce
     // a localized error message:
-    m_locale.AddCatalog("wxstd");
+    m_locale.AddCatalog("wxstd-3.0");
         // NOTE: it's not an error if we couldn't find it!
 
     // this catalog is installed in standard location on Linux systems and
diff -up wxWidgets-3.1.4/src/common/translation.cpp.1~ wxWidgets-3.1.4/src/common/translation.cpp
--- wxWidgets-3.1.4/src/common/translation.cpp.1~	2020-07-22 19:20:06.000000000 +0200
+++ wxWidgets-3.1.4/src/common/translation.cpp	2020-07-23 18:58:29.552872160 +0200
@@ -1522,7 +1522,7 @@ wxArrayString wxTranslations::GetAvailab
 
 bool wxTranslations::AddStdCatalog()
 {
-    if ( !AddCatalog(wxS("wxstd")) )
+    if ( !AddCatalog(wxS("wxstd-3.0")) )
         return false;
 
     // there may be a catalog with toolkit specific overrides, it is not
