--- wxWidgets-3.0.2/src/common/appbase.cpp.orig	2016-02-02 22:06:48.213958927 +0000
+++ wxWidgets-3.0.2/src/common/appbase.cpp	2016-02-02 22:07:01.738789843 +0000
@@ -750,7 +750,6 @@
 #if 0 // can't use wxLogTrace, not up and running yet
     printf("checking build options object '%s' (ptr %p) in '%s'\n",
              optionsSignature, optionsSignature, componentName);
-#endif
 
     if ( strcmp(optionsSignature, WX_BUILD_OPTIONS_SIGNATURE) != 0 )
     {
@@ -767,7 +766,7 @@
         // normally wxLogFatalError doesn't return
         return false;
     }
-
+#endif
     return true;
 }
 
